auth-api/src/auth_api/services/entity.py

```diff
@@ -31,7 +31,9 @@
 from auth_api.utils.passcode import passcode_hash
 from auth_api.utils.roles import ALL_ALLOWED_ROLES, Role
 from auth_api.utils.util import camelback2snake
+from .activity_log_publisher import publish_activity
 from .authorization import check_auth
+from ..utils.enums import ActivityAction
```

Comment:
relative import, we need to put some standards, either allow relative import nor not allow at all. In pay and legal-api, relative import is not allowed.